packages/react-frontend/components/Review.css

```diff
@@ -0,0 +1,32 @@
+.userReview {
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  padding: 0.5em;
+  margin: 1em;
+  width: 100%;
+}
+
+.userReview img {
+  height: 120px;
+  width: 120px;
+}
+
+.modifyIcons {
+  float: right;
+}
+
+.modifyIcons > * {
+  padding: 0.3em;
+}
+
+.modifyIcons {
+  color: white;
+  display: flex;
+}
+.reviewContent {
+  margin-left: 1em;
+}
+
```

Comment:
```suggestion
.reviewContent h3 {
  word-wrap: break-word;
}
```

Reply:
Okay I will look into adding the word-wrap.